docs/install.rst

```diff
@@ -251,23 +249,15 @@ Development Repository
 The latest development version of ``astropy`` can be cloned from GitHub
 using this command::
 
-   git clone --recursive git://github.com/astropy/astropy.git
+   git clone git://github.com/astropy/astropy.git
```

Comment:
So does the version numbering work for users too, without the extra git stuff needed in `.travis.yml`?

Reply:
Yes, it's just that Travis does a shallow clone by default